### ISSUE TITLE
Arduino Nano + MCP2515

### ISSUE THREAD

**Issue by osmancitci** [fmiglior/ETS2-and-ATS-can-dashboard#1]
I just started arduino nano and MCP2515 enough? If I buy a dashboard, which pins of the dashboard will I connect the signal connections to?

**Comment by fmiglior** [comment]
Yes. A arduino nano or uno and a mcp2515 is enough. You need to check the pinout os the dashboard. And find the Can high and Can low of the dashboard and connect it to the can high and can low of the mcp2515
________________________________
De: Osman Çitçi <notifications@github.com>
Enviado: sábado, 13 de fevereiro de 2021 14:42
Para: fmiglior/ETS2-and-ATS-can-dashboard <ETS2-and-ATS-can-dashboard@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Assunto: [fmiglior/ETS2-and-ATS-can-dashboard] Arduino Nano + MCP2515 (#1)


I just started arduino nano and MCP2515 enough? If I buy a dashboard, which pins of the dashboard will I connect the signal connections to?

**Comment by osmancitci** [comment]
So does every brand of dashboard panel provide support? Isn't it sends the same data? I want the one with lcd panel on it, isn't it possible to see the route information?

**Comment by fmiglior** [comment]
Every brand use can bus protocol. But they send different infos at different speeds. You can find on the internet the can bus speed and the data layout of the brand you want to use, you will need to adapt the code. Normally bmw, audi, vw is much easy to find the information.
I don’t know if the game send gps route info on the shared data. If it send is possibles.

________________________________
De: Osman Çitçi <notifications@github.com>
Enviado: domingo, 14 de fevereiro de 2021 07:13
Para: fmiglior/ETS2-and-ATS-can-dashboard <ETS2-and-ATS-can-dashboard@noreply.github.com>
Cc: fmiglior <fmiglior@hotmail.com>; Comment <comment@noreply.github.com>
Assunto: Re: [fmiglior/ETS2-and-ATS-can-dashboard] Arduino Nano + MCP2515 (#1)


So does every brand of dashboard panel provide support? Isn't it sends the same data? I want the one with lcd panel on it, isn't it possible to see the route information?

**Comment by osmancitci** [comment]
The data in the cancode.h file are just these data pieces

**Comment by fmiglior** [comment]
Is the part for the Citroen dashboard. The part I need to make the lights, speed and rpm work
For other brand will be different codes
________________________________
De: Osman Çitçi <notifications@github.com>
Enviado: domingo, 14 de fevereiro de 2021 13:25
Para: fmiglior/ETS2-and-ATS-can-dashboard <ETS2-and-ATS-can-dashboard@noreply.github.com>
Cc: fmiglior <fmiglior@hotmail.com>; Comment <comment@noreply.github.com>
Assunto: Re: [fmiglior/ETS2-and-ATS-can-dashboard] Arduino Nano + MCP2515 (#1)


The data in the cancode.h file are just these data pieces
